bitrise.yml

```diff
@@ -0,0 +1,98 @@
+format_version: 5
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+app:
+  envs:
+  # if you want to share this step into a StepLib
+  - MY_STEPLIB_REPO_FORK_GIT_URL: $MY_STEPLIB_REPO_FORK_GIT_URL
```

Comment:
we do not need the share related envs: MY_STEPLIB_REPO_FORK_GIT_URL & STEP_VERSION